src/BigQuery/BigQueryClient.php

```diff
@@ -183,78 +183,46 @@ public function __construct(array $config = [])
      *           of results. Setting this flag to a small value such as 1000 and
      *           then paging through results might improve reliability when the
      *           query result set is large.
-     *     @type array $defaultDataset Specifies the default datasetId and
-     *           projectId to assume for any unqualified table names in the
-     *           query. If not set, all table names in the query string must be
-     *           qualified in the format 'datasetId.tableId'.
+     *     @type int $startIndex Zero-based index of the starting row.
      *     @type int $timeoutMs How long to wait for the query to complete, in
      *           milliseconds. **Defaults to** `10000` milliseconds (10 seconds).
      *     @type int $maxRetries The number of times to retry, checking if the
      *           query has completed. **Defaults to** `100`.
-     *     @type bool $useQueryCache Whether to look for the result in the query
-     *           cache.
-     *     @type bool $useLegacySql If set to true the query will use
```

Comment:
We still support legacy :). The arguments have just changed here because we are no longer using jobs.query and instead are using jobs.insert.

It would look like the following:

```php
$queryResults = $bigQuery->runQuery($query, [
    'jobConfig' => [
        'useLegacySql' => true
    ]
]);
```